Topic1_Arrays/Day2307/TrungHau/1.txt

```diff
@@ -0,0 +1,27 @@
+// Time complexity: O(n * 3 + log(n)) 
```

Comment:
Time complexity O(3n + log(n)) same as time complexity O(n)
In this question, space complexity is O(n) also